server/services/echo_service.go

```diff
@@ -46,7 +46,7 @@ func (s *echoServerImpl) Echo(ctx context.Context, in *pb.EchoRequest) (*pb.Echo
 		return nil, err
 	}
 	echoTrailers(ctx)
-	return &pb.EchoResponse{Content: in.GetContent()}, nil
+	return &pb.EchoResponse{Content: in.GetContent(), Severity: in.Severity}, nil
```

Comment:
Please use the Getter for `Severity`: `Severity: in.GetSeverity()`

Reply:
Added a getter and started using it.